app/src/main/java/org/eclipse/kuksa/testapp/databroker/JavaDataBrokerEngine.java

```diff
@@ -33,11 +33,13 @@
 import org.eclipse.kuksa.connectivity.databroker.request.UpdateRequest;
```

Comment:
Functional Finding:
Value Output does not work for Array types (e.g. Vehicle.Cabin.Infotainment.SmartphoneProjection.SupportedMode)

It will print the Array.toString() which basically outputs rubbish like [Ljava.lang.String;@1234abc]

Reply:
Supported string arrays for now. Most of the fixes need to be done in #109.